### ISSUE TITLE
Timestamps seem not to work

### ISSUE THREAD

**Issue by ataube** [calebmer/postgraphql#17]
Hi,
I played around with the forum example and added some timestamps. It seems like the postgraphql server is not able to read them. The query throws the following error:

```javascript
//query
{
  post(id: "cG9zdDox") {
    id
    createdAt
  }
}

//result
{
  "data": {
    "post": {
      "id": "cG9zdDox",
      "createdAt": null
    }
  },
  "errors": [
    {
      "message": "Cannot read property '_scalarConfig' of undefined",
      "stack": "TypeError: Cannot read property '_scalarConfig' of undefined\n    at serialize (/workspace/postgraphql/examples/forum/node_modules/graphql/type/definition.js:170:28)\n    at GraphQLScalarType.serialize (/workspace/postgraphql/examples/forum/node_modules/graphql/type/definition.js:171:14)\n    at completeLeafValue (/workspace/postgraphql/examples/forum/node_modules/graphql/execution/execute.js:606:37)\n    at completeValue (/workspace/postgraphql/examples/forum/node_modules/graphql/execution/execute.js:560:12)\n    at completeValueCatchingError (/workspace/postgraphql/examples/forum/node_modules/graphql/execution/execute.js:477:21)\n    at resolveField (/workspace/postgraphql/examples/forum/node_modules/graphql/execution/execute.js:450:10)\n    at /workspace/postgraphql/examples/forum/node_modules/graphql/execution/execute.js:256:18\n    at Array.reduce (native)\n    at executeFields (/workspace/postgraphql/examples/forum/node_modules/graphql/execution/execute.js:254:50)\n    at completeObjectValue (/workspace/postgraphql/examples/forum/node_modules/graphql/execution/execute.js:648:10)\n    at completeValue (/workspace/postgraphql/examples/forum/node_modules/graphql/execution/execute.js:571:12)\n    at /workspace/postgraphql/examples/forum/node_modules/graphql/execution/execute.js:524:14\n    at process._tickCallback (node.js:382:9)"
    }
  ]
}

//post table
create table post (
  id               serial not null primary key,
  author_id        int not null references person(id),
  headline         text not null,
  topic            post_topic,
  body             text,
  created_at       timestamp with time zone
);
```
Is this a know issue?

**Comment by calebmer** [comment]
Thanks for reporting, I'll take a look at this.

**Comment by calebmer** [comment]
This is fixed in version `1.1.3`, thanks again! 😄
